projects/pgai/pgai/cli.py

```diff
@@ -220,6 +323,7 @@ def cli():
 
 vectorizer.add_command(vectorizer_worker)
 vectorizer.add_command(download_models)
+vectorizer.add_command(vectorizer_worker_benchmark)
```

Comment:
What is the purpose of this?

Reply:
Can we move this functionality into the "benchmark" project?

Reply:
Moved it to the benchmark dir.